File: Python/URI 1019.py
N = int(input())

horas = N//3600
minutos = N//60 - (horas * 60)
segundos = N - (minutos * 60 + horas * 3600)

print("{}:{}:{}".format(horas,minutos,segundos))

# // = divisão inteira (é necessário usar para o resultado da dvisão não ser um float)

File: C/URI 1019.c
#include <stdio.h>
int main(void) {
  int N,horas,minutos,segundos;
  scanf("%d",&N);
  horas = N / 3600;
  minutos = N / 60 - (horas * 60);
  segundos = N - (minutos * 60 + horas * 3600); 
  printf("%d:%d:%d\n",horas,minutos,segundos);
  return 0;
}